fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/SumoAmbassador.java

```diff
@@ -109,28 +97,26 @@ public SumoAmbassador(AmbassadorParameter ambassadorParameter) {
         if (!absoluteConfiguration.exists()) {
             absoluteConfiguration = new File(this.ambassadorParameter.configuration.getParent(), sumoConfig.sumoConfigurationFile);
         }
+        sumoConfigurationFile = absoluteConfiguration;
 
-        routeFilename = parseRouteFilename(absoluteConfiguration);
-
-        if (StringUtils.isBlank(routeFilename)) {
-            String myError = "The tag <route-files value=\"*.rou.xml\"/> is missing in " + absoluteConfiguration.getAbsolutePath();
-            log.error(myError);
-            throw new RuntimeException(myError);
-        }
+        vehicleTypeRouteFileName = getVehicleTypeRouteFileName();
     }
 
     /**
      * Find the name of the route file and return it.
      *
-     * @param sumoConfigurationFile The SUMO configuration file.
-     * @return The route-files filename.
+     * @return The route-file name.
      */
-    private String parseRouteFilename(File sumoConfigurationFile) {
-
+    private String getVehicleTypeRouteFileName() {
         try {
             XMLConfiguration sumoConfiguration = XmlUtils.readXmlFromFile(sumoConfigurationFile);
-            String routeFile = XmlUtils.getValueFromXpath(sumoConfiguration, "/input/route-files/@value", null);
-            return Validate.notNull(routeFile);
+            String routeFileName = XmlUtils.getValueFromXpath(sumoConfiguration, "/input/route-files/@value", null);
+            if (routeFileName == null) {
+                return "routes_vTypes.xml";
```

Comment:
This will be the temporary route file name if the `<route-files /> `-tag is missing.  Is this fine?

Reply:
I would prefer to align to SUMO conventions, ending with `.rou.xml` ( e.g. `vtypes.rou.xml`)

Reply:
Oh yeah, I meant to use `.rou.xml` that was an oversight.